app/models/allocation_history.rb

```diff
@@ -175,9 +175,19 @@ def deallocate_offender(event:, event_trigger:)
     PomMailer.with(**mail_params).offender_deallocated.deliver_later
   end
 
-  def push_pom_to_community_api
-    if saved_change_to_primary_pom_nomis_id?
-      PushPomToDeliusJob.perform_later(self)
-    end
+  def publish_allocation_changed_event
+    return unless saved_change_to_primary_pom_nomis_id?
+
+    DomainEvents::Event.new(
+      event_type: 'offender-management.pom.changed',
```

Comment:
You don't add `offender-management.` - the class adds that automatically.

Also, is `pom` the change here? Or did the `pom-allocation` change? If you've already agreed the event type with the Delius team, then ignore me.

Reply:
Agree. `allocation.changed` would be better